composer.json

```diff
@@ -76,5 +87,11 @@
     "test": "phpunit",
     "test-coverage": "phpunit --coverage-clover build/logs/clover.xml",
     "docs": "bookdown docs/bookdown.json"
-  }
+  },
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/prolic/prophecy.git"
```

Comment:
Prophecy has currently no PHP 7.1 support.
I created a PR, but it's not yet accepted, see: https://github.com/phpspec/prophecy/pull/287